src/android/SimpleCameraPreview.java

```diff
@@ -78,6 +84,33 @@ public boolean execute(String action, JSONArray args, CallbackContext callbackCo
         }
     }
 
+    private boolean getRatio(JSONObject options, CallbackContext callbackContext) {
+        int targetSize = 0;
+        try {
+            if (options.getString("targetSize") != null && options.getString("targetSize") != "null") {
+                targetSize = Integer.parseInt(options.getString("targetSize"));
+            }
+        } catch (JSONException | NumberFormatException e) {
+            e.printStackTrace();
```

Comment:
should we send a callback error here, or continue normal execution ? @ombr @azharbeebeejaun , user would not be aware if the configuration passed is throwing an exception right ?

Reply:
We can skip this part for now and implement it when users complain or whenthis feature is more widely used.

Reply:
okay